js/filter-control.js

```diff
@@ -4,19 +4,32 @@ var $ = require('jquery');
 
 function FilterControl($selector) {
   this.$element = $selector;
+  this.formType = this.getFormType();
   this.modifiesFilter = this.$element.data('modifies-filter');
   this.modifiesProperty = this.$element.data('modifies-property');
 
-  this.controlledFilter = $('#' + this.$element.data('controls'));
   this.$element.on('change', this.handleChange.bind(this));
 }
 
+FilterControl.prototype.getFormType = function() {
+  if (this.$element.find('input[type=checkbox]').length > 0) {
+    return 'checkbox';
+  } else if (this.$element.find('select').length > 0) {
+    return 'select';
+  }
```

Comment:
Yep. It might be good to abstract this more down the road, but that's all we have at the moment.